src/main/java/quickndirty/minisurveymonkey/Response.java

```diff
@@ -0,0 +1,24 @@
+package quickndirty.minisurveymonkey;
+
+import quickndirty.minisurveymonkey.QuestionTypes.QType;
+
+public class Response {
+	protected int ID;
```

Comment:
I think that `Response` should have a reference attribute to the question it's answering.

Reply:
I'll add the question reference, but what do you mean its own ID?  You mean like the associated Question's ID?  Response should already have its own ID

Reply:
Just a reference I assume, like how the AddressBook and BuddyInfo were setup, since it's going to be a one to many relationship. This way spring can take care of the association once its annotated.

Reply:
Okay, Question reference is added